crates/ra_assists/src/handlers/add_missing_impl_members.rs

```diff
@@ -188,6 +191,14 @@ fn add_missing_impl_members_inner(
     })
 }
 
+fn remove_bounds(ty_def: ast::TypeAliasDef) -> ast::TypeAliasDef {
```

Comment:
Lets move this to `edit.rs`, see `ast::TypeParam::remove_bounds`

Reply:
Oh yeah that's way better